apps/web/app/modules/Admin/Admin.layout.tsx

```diff
@@ -28,7 +30,9 @@ export const clientLoader = async () => {
   return null;
 };
 
-const AdminGuard = ({ children }: { children: React.ReactNode }) => {
+type AdminGuardProps = PropsWithChildren;
+
+const AdminGuard = ({ children }: AdminGuardProps) => {
```

Comment:
```suggestion
const AdminGuard = ({ children }: PropsWithChildren) => {
```
if there is no other props can we keep it simple?